src/pages/settings/Settings.tsx

```diff
@@ -166,6 +171,15 @@ const Settings: FC = () => {
       >
         <NotificationRow />
         <Row>
+          {!metadataConfigurationSupport && (
```

Comment:
any suggestions for better copy?

Reply:
Maybe `hasMetadataConfigApi`?

All of the neighbours could be `hasFooApi` instead of `fooSupport`, do you think that is better?

Reply:
aah sorry if I was unclear, I meant for the notification text content and not the variable name. I tried to align these variable names with the actual api extension name, so maybe we can do `hasMetadataConfiguration` instead and similar pattern for the others? api extension can be related to sub features of an api endpoint so I think the `hasApi` pattern might be a bit ambiguous.

Reply:
Right I agree with variable naming either way.

As for the notification copy, the text seems OK, maybe add "or later" and "server" to the string:

"Update to the LXD v5.19.0 or later to access more server settings"

Reply:
Yeah that sounds good, will add :+1: